include/pybind11/stl_bind.h

```diff
@@ -364,6 +364,36 @@ NAMESPACE_BEGIN(detail)
 /* Fallback functions */
 template <typename, typename, typename... Args> void map_if_insertion_operator(const Args&...) { }
 
+template <typename Map, typename Class_, typename... Args> void map_if_const_value(Class_ &cl, const Args&...)
+{
```

Comment:
Indentation issue. Also, this PR suspiciously looks like it's using tabs instead of spaces ;)